src/components/MenuBar/index.tsx

```diff
@@ -4,65 +4,51 @@ import WifiIcon from '@/assets/icons/WifiIcon.svg?react';
 import ControlCenterIcon from '@/assets/icons/ControlCenterIcon.svg?react';
 import MenuBarItem from './MenuBarItem';
 import Clock from './Clock';
-import useToggleMenu from './useToggleMenu';
 import MenuList from './MenuList';
-import useOverlay from '@/hooks/useOverlay';
-import Alert from '../Alert';
 import MenuOverlay from './MenuOverlay';
-import { useState } from 'react';
 import ControlCenter from './ControlCenter';
 import useAudio from '@/hooks/useAudio';
 import MusicSrc from '@/assets/music/sample.mp3';
-
-type MenuType = 'menu' | 'controlCenter';
+import type { MouseEvent } from 'react';
+import useOverlay from '@/hooks/useOverlay';
 
 const DEFAULT_MENUS = ['Menu1', 'Menu2', 'Menu3', 'LongTextMenuLongTextMenuLongTextMenu'];
 
 const MenuBar = () => {
-  const { isOpen, overlayRef, overlayPosition, openMenu } = useToggleMenu();
-
-  const { isPlaying, togglePlay } = useAudio({ src: MusicSrc });
-
-  const [showMenuType, setShowMenuType] = useState<'menu' | 'controlCenter'>('menu');
+  const { togglePlay } = useAudio({ src: MusicSrc });
 
   const overlay = useOverlay();
 
-  const handleMenuBarItemClick = (event: React.MouseEvent<HTMLButtonElement>, menuType: MenuType) => {
+  const openMenu = (event: MouseEvent<HTMLButtonElement>) => {
     event.stopPropagation();
     const rect = event.currentTarget.getBoundingClientRect();
-    openMenu(rect);
-    setShowMenuType(menuType);
+    overlay.open(() => (
+      <MenuOverlay initialPosition={{ x: rect.x, y: rect.y + rect.height }} close={overlay.close}>
+        <MenuList menus={DEFAULT_MENUS} />
+      </MenuOverlay>
+    ));
   };
 
-  const openOverlay = () => {
-    overlay.open(
-      <Alert
-        appIconUrl='src/assets/icons/Dock/Safari_Icon.png'
-        title='Title'
-        description='Description text about this alert is shown 
-          here, explaining to users what the options 
-          underneath are about and what to do.'
-      >
-        <Alert.Button fill onClick={overlay.close}>
-          Close Button
-        </Alert.Button>
-        <Alert.Button>Dummy Button</Alert.Button>
-      </Alert>
-    );
+  const openControlCenter = (event: MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
+    const rect = event.currentTarget.getBoundingClientRect();
+    overlay.open(() => (
+      <MenuOverlay initialPosition={{ x: rect.x, y: rect.y + rect.height }} close={overlay.close}>
+        <ControlCenter togglePlay={togglePlay} />
+      </MenuOverlay>
```

Comment:
```suggestion
      <MenuOverlay initialPosition={{ x: rect.x, y: rect.y + rect.height }} close={overlay.close}>
        <ControlCenter />
      </MenuOverlay>
```

마찬가지로 Audio 관련 상태는 `ControlCenter` 내부에서 관리하게 된다면 따로 `prop`을 내려 줄 필요는 없어질 것 같습니다. 😀